.buildkite/al2env.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+shim_base=/tmp/shim-base
+uuid=$BUILDKITE_BUILD_NUMBER
+dir=$shim_base/$uuid
```

Comment:
How about using $BUILDKITE_BUILD_NUMBER directly instead of putting that in $uuid? I would assume $uuid is hex-number-with-dashes, but this variable is not.

Reply:
I changed it to `uid`. I much prefer that, since that signifies what it is suppose to be, a unique ID.

Reply:
`uid`, in the context of Linux, would generally imply a User ID, not a unique ID.

Reply:
Yea, there is some ambiguity there, perhaps `uniqueID`?

Reply:
unique_id since other variables are snake_case?

Reply:
Fixed!